index.html

```diff
@@ -263,6 +263,23 @@ <h3>CSS</h3>
     </section>
     <!-- ─── End About Myself ────────────────────────────────────────── -->
 
+    <!-- ─── Contact Me ──────────────────────────────────────────────── -->
+    <section class="contact">
+        <div class="contact-info">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/mdovkzyl" method="POST" class="contact-form">
+            <input type="text" name="name" id="name" placeholder="Cosmos Hagan" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="cosmoshagan@gmail.com" required>
+            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            <div class="action">
+                <button type="submit" class="btn">Get In Touch</button>
```

Comment:
- [ ] You did a good job styling all the buttons inside the document :+1: :100:, but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) you should add `:hover` and `:active` interactions to the design of your project because it will improve the user experience :+1: 

**Figma design**
![Screenshot from 2023-01-14 07-40-42](https://user-images.githubusercontent.com/105117832/212472166-701382c9-9303-48fb-b0b5-e2a94147dcb6.png)